Makefile

```
@@ -15,3 +15,6 @@ docs-install:
 docs-dev:
 	cd ./docs && npm start
 
+schema-gen:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)